src/components/CatalogueItemData.tsx

```diff
@@ -60,25 +82,46 @@ const CatalogueItemData = ({ catalogueItem }: Props) => {
 				</span>
 				{datasets.map(dataset => (
 					<div className='flex flex-row items-center' key={dataset.description}>
-						<div className='w-2/3'>
-							<a
-								href={dataset.url}
-								key={`${dataset.description}`}
-								target='_blank'
-								rel='noreferrer'
-								className='hover:underline'
-							>
-								{dataset.description}
-							</a>
-							<span className='ml-3 rounded-3xl bg-gray-200 py-1 px-2 text-xs font-semibold text-gray-600'>
-								{dataset.type.includes('raw')
-									? getFileFormat(dataset.type, 'dataset-raw-').toUpperCase()
-									: getFileFormat(dataset.type, 'dataset-').toUpperCase()}
-							</span>
+						<div>
+							<div className='mb-5 text-sm font-bold text-gray-700'>
+								{dataset.name}
+							</div>
+							<div className='mb-3'>
+								<a
+									href={dataset.url}
+									key={`${dataset.description}`}
+									target='_blank'
+									rel='noreferrer'
+									className='hover:underline'
+								>
+									{dataset.description}
+								</a>
+							</div>
+							<div className='flex'>
+								{dataset['alt-format'].map((altFormat, index) => (
+									<div key={index} className='mr-2 mb-4'>
+										<a
+											href={TEMP_REDIRECT_FORM}
+											key={`${dataset.name}`}
```

Comment:
I think you should use `key=${altFormat.type}` instead of `${dataset.name}` since `dataset.name` will not be unique across the different formats (but type will be unique since that's what distinguishes the different alt-formats)
You also might want to add something like
```
href={`${TEMP_REDIRECT_FORM}?item=${catalogueItem.id}&url=${encodeURIComponent(altFormat.url)}`}
```
So that the redirected form will have the item id and the url to forward to